src/spikeinterface/core/job_tools.py

```diff
@@ -136,11 +136,8 @@ def divide_segment_into_chunks(num_frames, chunk_size):
     else:
         n = num_frames // chunk_size
 
-        frame_starts = np.arange(n) * chunk_size
-        frame_stops = frame_starts + chunk_size
-
-        frame_starts = frame_starts.tolist()
-        frame_stops = frame_stops.tolist()
+        frame_starts = [i * chunk_size for i in range(n)]
+        frame_stops = [(i + 1) * chunk_size for i in range(n)]
```

Comment:
This works. I tend to lean toward explicit so something like

```

frame_stops = [frame_start + chunk_size for frame_start in frame_starts]
```

Just because it is clearer rather than have to multiple things out. But I think this is a good protection against overflow. Your long recordings are really finding all the overflows :)

Reply:
Yeah... I am surprised no one has tried to deal with long recordings before. Are there any other long-recording users?

Reply:
The most I've heard of recently is closer to the 10-15 hour mark. But haven't really followed up on those. These things work until they don't. So we are trying to track them down, but they are easier to find when they break.

Reply:
Hope that long recording is not a bad idea :)

Reply:
@jiumao2 we are planning to run week long recordings. You are in good company :)

Reply:
Good to hear that! Not alone anymore :)

Reply:
Hi @alejoe91 ,

Are you beginning to work with the week-long recordings? I'm not satisfied with the results from Kilosort and have run into a number of issues. Could you share any strategies or tools you're using, such as different sorters, that might be helpful?

Thank you!